webClient/src/app/components/filebrowseruss/filebrowseruss.component.html

```diff
@@ -0,0 +1,85 @@
+
```

Comment:
This is a file that got left over from "webClient" folder. **src-for-widgets** (widget version of FE) doesn't use webClient, just have to delete this file